libs/service-portal/finance/src/screens/PaymentOverview/PaymentOverview.tsx

```diff
@@ -13,7 +13,7 @@ export const PaymentOverview: ServicePortalModuleComponent = () => {
         defaultMessage: 'Greiðslur',
       })}
       intro={defineMessage({
-        id: 'sp.assets:intro',
+        id: 'sp.payments:intro',
```

Comment:
`useNamespaces('sp.payments')` is never run.

shouldn't this be `sp.finance:*` ??

Reply:
This module is actually not being used for now so it will default to the defaultMessage.
Not sure if this should be under sp.finance, but it certainly does not fit under sp.assets so I changed it.. 

This will need to be fixed once this module gets back into the project.